src/server/qgis_map_serv.cpp

```diff
@@ -39,7 +41,24 @@ int fcgi_accept()
 
 int main( int argc, char *argv[] )
 {
-  QgsApplication app( argc, argv, getenv( "DISPLAY" ), QString(), QStringLiteral( "server" ) );
+  // Test if the environ variable DISPLAY is defined
+  // if it's not, the server is running in offscreen mode
+  // Qt supports using various QPA (Qt Platform Abstraction) back ends
+  // for rendering. You can specify the back end to use with the environment
+  // variable QT_QPA_PLATFORM when invoking a Qt-based application.
+  // Available platform plugins are: directfbegl, directfb, eglfs, linuxfb,
+  // minimal, minimalegl, offscreen, wayland-egl, wayland, xcb.
+  // http://doc.qt.io/qt-5/qpa.html
```

Comment:
Thanks for the documentation. Where is the list from? I can't find it in the qpa doc, is it from stackoverflow?

Reply:
I found it here https://www.ics.com/blog/qt-tips-and-tricks-part-1

Reply:
That's the link I was expecting to have in the comment. Finding the QPA docs should be straightforward for all the devs.

Reply:
Ok, I'll update the comment

Reply:
Thanks, merge away